modules/monitoring/public/css/tables.less

```diff
@@ -280,3 +280,7 @@
     }
   }
 }
+
+.plugin-output .output-table {
+    font-size: 0.75em;
+}
```

Comment:
Sorry, but now I think this fits better in the `.plugin-output` ruleset in `module.less`. There's already the other rules for states.